stdlib/jdbc/src/test/java/org/ballerinax/jdbc/actions/UpdateTest.java

```diff
@@ -229,12 +229,24 @@ public void testInsertBoolDataAsIntsAndReturnInts() {
         Assert.assertEquals(((BInteger) returns[2]).intValue(), 1);
     }
 
+    /**
+     * Test for getting auto-generated key when getGeneratedKey is true and the table doesn't have auto generated key.
```

Comment:
Instead of this comment, shall we use descriptive name for the function itself. This is the pattern followed for other test cases as well.